### ISSUE TITLE
json_query filter moved to community.general collection

### ISSUE THREAD

**Issue by lapecoravolante** [ownport/portable-ansible#24]
Can you include this collection in the portable zip?

**Comment by FBnil** [comment]
Adding it yourself is not difficult. json_query is build on top of jmespath and that's not installed either.

use "pip3 install jmespath" to install it to ~/.local/lib/python3.*/site-packages/jmespath
But due to the isolated walled nature of portable-ansible, it's not "seen".
So copy that jmespath into the portable-ansible directory (so that it's in the same directory as paramiko), or, as the documentation of portable-ansible says, put it in ./portable-ansible/extras/
I tested with jmespath 1.0.1 (wget) and 0.10.0 (pip3)
Now, next to your playbook, create a directory called filter_plugins and drop in there the json_query.py
Or use ANSIBLE_FILTER_PLUGINS environment variable to a directory with your filter(s), or configure [ansible.cfg](https://docs.ansible.com/ansible/latest/reference_appendices/config.html#ansible-configuration-settings).

So in short:
```
# check https://github.com/jmespath/jmespath.py/tags
cd portable-ansible/extras
wget -O jmespath.tgz https://github.com/jmespath/jmespath.py/archive/refs/tags/1.0.1.tar.gz 
tar xf jmespath.tgz --transform='s|jmespath.py-1.0.1|.|'  jmespath.py-1.0.1/jmespath/
echo "Manually run: rm $(pwd)/jmespath.tgz"
cd -
mkdir filter_plugins
cd filter_plugins
wget https://github.com/ansible-collections/community.general/raw/main/plugins/filter/json_query.py
cd -
```
You can now repackage the portable-ansible with the added code.

I agree that the size is so small, that jmespath could be added at minimal size cost. (just by adding it to the requirements file before building)

**Comment by lapecoravolante** [comment]
I've solved by downloading the entire community.general collection and shipping it with the ansible zip.
Thanks.

**Comment by FBnil** [comment]
hmm, maybe we should document that in the main page, as in https://github.com/ansible-collections/community.general/tree/main/plugins we see a lot of oldies (that were previously in portable-ansible, and now are not)
